drush/Commands/SanitizationCommands.php

```diff
@@ -49,6 +49,14 @@ public function sanitizeEntities($result, CommandData $commandData) {
       return;
     }
     $types = array_keys($this->entityTypeManager->getDefinitions());
+
+    // Deleting menu link content sometimes results in rendering issues with
+    // super-sanitized databases, so excluding it from sanitization.
+    $idx = array_search('menu_link_content', $types);
+    if ($idx !== -1) {
```

Comment:
Why the -1 check in addition to the array_search?

Reply:
Mostly for consistency's sake in the end. See line [63](https://github.com/massgov/openmass/pull/597/files#diff-fa3dc0c7a267917545b458fa2f1bb1d7306f4da39e4199aa8b9ffd43455a6d3aR63) below this. I tried looking into why that check might be there as well, and my best educated guess is that they wanted to verify that the result was actually true and not set explicitly to -1 either out of an abundance of caution or because they ran into an issue (though I cannot imagine a situation where this would actually occur), since array_search will evaluate -1 as true.

Reply:
OK